app/uk.gov.hmrc.helptosavestridefrontend/models/register/CreateAccountRequest.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavestridefrontend.models.register
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.helptosavestridefrontend.models.NSIUserInfo
+
+case class CreateAccountRequest(userInfo: NSIUserInfo, eligibilityReason: Option[Int])
```

Comment:
in stride-frontend are we ever expecting the eligibility reason not to be populated? i.e. does `eligibilityReason` need to be an `Option` here?